cmd/tfplugindocs/testdata/scripts/schema-json/migrate/time_provider_success_docs_website.txtar

```diff
@@ -643,6 +652,25 @@ $ terraform import time_static.example 2020-02-12T06:36:13Z
 ```
 
 The `triggers` argument cannot be imported.
+-- docs/list-resources/example.html.markdown --
+---
+layout: "time"
+page_title: "Time: time_example"
+description: |-
+  Just a fake example.
+---
+
+Just a fake example.
+
+```hcl
```

Comment:
Similar to the other since migrate it likely just copying anything over, I think this code block would be `terraform`